### PR TITLE
Added build script for sphinx_rtd_theme

### PR DESCRIPTION
**Pull request by saurabh-gore**
- [x] Have you checked and followed all the points mention in the [CONTRIBUTING.MD](https://github.com/ppc64le/build-scripts/blob/master/CONTRIBUTING.md)
- [x] Did you run the script(s) on fresh container with `set -e` option enabled and observe success ?

Package Name : sphinx_rtd_theme
Package Version : 0.5.1
Package URL : [sphinx_rtd_theme](https://github.com/readthedocs/sphinx_rtd_theme)

Tested On : UBI 8.5

### CODE REVIEW

File: s/sphinx_rtd_theme/sphinx_rtd_theme_ubi_8.5.sh [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+set -e
+
+#Variables
+PACKAGE_NAME=sphinx_rtd_theme
+PACKAGE_VERSION="${1:-0.5.1}"
+PACKAGE_URL=https://github.com/readthedocs/sphinx_rtd_theme.git
+
+#Install dependencies.
+yum install -y git python2 python2-devel python3 python3-devel gcc-c++ make
```
Comment (Siddhesh-Ghadi) [inline s/sphinx_rtd_theme/sphinx_rtd_theme_ubi_8.5.sh:28]: Is `python2` required? I see use of `python3` in script.

**Review by Siddhesh-Ghadi — CHANGES_REQUESTED** [review]

File: s/sphinx_rtd_theme/sphinx_rtd_theme_ubi_8.5.sh (hunk 1 above)
Comment (saurabh-gore) [reply s/sphinx_rtd_theme/sphinx_rtd_theme_ubi_8.5.sh:28]: @Siddhesh-Ghadi  Yes , Python2 is required for building package.

Comment (Siddhesh-Ghadi) [reply s/sphinx_rtd_theme/sphinx_rtd_theme_ubi_8.5.sh:28]: Okay

**Review by Siddhesh-Ghadi — APPROVED** [review]
LGTM

**Pull request merged**